packages/block-library/src/navigation/style.scss

```diff
@@ -41,8 +41,6 @@ $navigation-icon-size: 24px;
 	}
 
 	// Menu item link.
-	// By adding low specificity, we enable compatibility with link colors set in theme.json,
-	// but still allow them to be overridden by user-set colors.
 	.wp-block-navigation-item__content {
 		color: inherit;
```

Comment:
I think we should remove this line. `.wp-block-navigation-item__content` is a class that is used on a link. This line forces links to inherit their color from their parent which is against the pattern of CSS.